public/components/integrations/components/setup_integration.tsx

```diff
@@ -60,9 +61,9 @@ const INTEGRATION_CONNECTION_DATA_SOURCE_TYPES: Map<
   [
     's3',
     {
-      title: 'Table',
-      lower: 'table',
-      help: 'Select a table to pull the data from.',
+      title: 'Catalog',
```

Comment:
Why catalog here?

Reply:
Because the form isn't referring to a table, I asked around for a better name.